### PR TITLE
Update replace-license-key.adoc - updated windows options to get the base64 string

### PR DESCRIPTION
**Pull request by steftux**
updated windows options to get the base64 string

# Writer's Quality Checklist

Before merging your PR, did you:

- [x ] Run spell checker
- [x ] Run link checker to check for broken xrefs
- [x ] Check for orphan files
- [x ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [x ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [x ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/replace-license-key.adoc [hunk 1]
```diff
@@ -13,42 +13,47 @@ this node is named `rtf-controller-1` by default.
 
 The `rtfctl` utility is required to replace your Mule license key on Runtime Fabric. Follow the steps in xref:install-rtfctl.adoc[Install rtfctl] before continuing.
 
+For Runtime Fabric on VMs / Bare Metal, the `rtfctl` binary is in directory `/opt/anypoint/runtimefabric/`.
+
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/replace-license-key.adoc:17]: @kmendiratta-crm Can you confirm that this is the correct additions you asked me to add?

**Comment by IsaacEldridge** [comment]
Thanks Stefan. I just wanted to add something about running rtfctl commands, and I'm waiting on the eng manager to confirm. I'll merge this up as soon as I'm done with that. Thanks for the contribution!

**Comment by IsaacEldridge** [comment]
@kmendiratta-crm please confirm or suggest changes to these additions I added, so we can get this merged. Thanks!

File: modules/ROOT/pages/replace-license-key.adoc [hunk 2]
```diff
@@ -13,42 +13,47 @@ this node is named `rtf-controller-1` by default.
 
 The `rtfctl` utility is required to replace your Mule license key on Runtime Fabric. Follow the steps in xref:install-rtfctl.adoc[Install rtfctl] before continuing.
 
+For Runtime Fabric on VMs / Bare Metal, the `rtfctl` binary is in directory `/opt/anypoint/runtimefabric/`.
```
Comment (kmendiratta-crm) [inline modules/ROOT/pages/replace-license-key.adoc]:
```suggestion
For Runtime Fabric on VMs / Bare Metal, the `rtfctl` binary is in directory `/opt/anypoint/runtimefabric/` and the current working directory is also `/opt/anypoint/runtimefabric/`.
```

**Review by dhirsch1 — DISMISSED** [review]

File: modules/ROOT/pages/replace-license-key.adoc [hunk 3]
```diff
@@ -13,42 +13,47 @@ this node is named `rtf-controller-1` by default.
...
-base64 -w0 license.lic
+BASE64_ENCODED_LICENSE=$(base64 -w0 license.lic)
 ----
-** On Windows, a shell terminal emulator (such as cygwin) or access to a Unix-based 
-computer is required.
-
-... Transfer to your Unix environment if necessary.
-... Run the following command to Base64 encode the license key:
+** On Windows, choose one of the following: 
+*** Use a WSL or Cygwin shell that includes the base64, tool and use the above Unix command.
```
Comment (dhirsch1) [inline modules/ROOT/pages/replace-license-key.adoc]: Should there be a comma between base64 and tool?

Comment (IsaacEldridge) [inline modules/ROOT/pages/replace-license-key.adoc]:
```suggestion
*** Use a WSL or Cygwin shell that includes the base64 tool and use the above Unix command. 
```

**Review by dhirsch1 — APPROVED** [review]

**Review by kmendiratta-crm — COMMENTED** [review]
LGTM

**Pull request merged**